GetSubscriptionsFeed/__tests__/handler.test.ts

```diff
@@ -0,0 +1,121 @@
+jest.mock("winston");
+
+import { TableService } from "azure-storage";
+import { MaxAllowedPaymentAmount } from "io-functions-commons/dist/generated/definitions/MaxAllowedPaymentAmount";
+import { toAuthorizedCIDRs } from "io-functions-commons/dist/src/models/service";
+import {
+  IAzureApiAuthorization,
+  UserGroup
+} from "io-functions-commons/dist/src/utils/middlewares/azure_api_auth";
+import { IAzureUserAttributes } from "io-functions-commons/dist/src/utils/middlewares/azure_user_attributes";
+import { NonNegativeNumber } from "italia-ts-commons/lib/numbers";
+import {
+  EmailString,
+  NonEmptyString,
+  OrganizationFiscalCode
+} from "italia-ts-commons/lib/strings";
+
+import { GetSubscriptionsFeedHandler } from "../handler";
+import { TableEntry } from "../utils";
+
+afterEach(() => {
+  jest.resetAllMocks();
+  jest.restoreAllMocks();
+});
+
+const anOrganizationFiscalCode = "01234567890" as OrganizationFiscalCode;
```

Comment:
If it is useful and will be used in other tests please create a new shared file.

Reply:
shared file in this repo? In case can I put it in the utils folder? Any different conventions?

Reply:
In a __mocks__ directory like https://github.com/teamdigitale/io-functions-admin/blob/master/__mocks__/mocks.ts